src/Network/Eureka/Version/Cabal.hs

```
@@ -3,14 +3,20 @@ module Network.Eureka.Version.Cabal (
   , fromVersion
   , fromString
   , showVersion
+  , versionInRangeP
   ) where
 
-import           Data.Maybe                   (listToMaybe)
-import qualified Data.Version                 as DV (Version, Version (Version),
-                                                     parseVersion, showVersion)
-import           Network.Eureka.Version.Types (Version)
-import           Text.ParserCombinators.ReadP (readP_to_S)
+import Data.Maybe (listToMaybe)
+import qualified Data.Version as DV (Version, Version (Version),
+  parseVersion, showVersion)
+import Network.Eureka.Version.Types (Version)
+import Text.ParserCombinators.ReadP (readP_to_S)
+import Network.Eureka.Version.Types (Predicate)
 
+versionInRangeP :: [Int] -> [Int] -> Predicate
```

Okay. Is it an error case if the two lists provided are of different lengths? If so, should we fail fast and explode?